src/components/SentimentAnalysisGraph.js

```diff
@@ -1,44 +1,40 @@
 import React, {Component} from 'react';
 import {VictoryAxis, VictoryBar, VictoryChart} from 'victory';
 
-const width = 30;
-const barFontSize = 7;
-const axisFontSize = 25;
+const totalValue = 100;
 export default class SentimentAnalysisGraph extends Component {
 	static displayName = 'SentimentAnalysisGraph';
 
 	static propTypes = {
-		posValue: React.PropTypes.number,
-		negValue: React.PropTypes.number,
-		barHeight: React.PropTypes.number
+		posValue: React.PropTypes.number
 	};
 
 	static defaultProps = {
-		posValue: 90,
-		negValue: 10,
-		barHeight: 250
+		posValue: 68
 	};
 
-	render() {
-		return (
-			<VictoryChart domain={{y: [0, 100]}} height={this.props.barHeight}>
+	render(){
```

Comment:
`render() {...` -- should always have a space space between function parens and curlies